cmd/cli/plugin/package/package_available_get.go

```diff
@@ -132,6 +132,9 @@ func getValuesSchema(cmd *cobra.Command, args []string, kc kappclient.Client) er
 	}
 
 	var parseErr error
+	if len(pkg.Spec.ValuesSchema.OpenAPIv3.Raw) == 0 {
+		return errors.Errorf("package '%s/%s' does not have any user configurable values in the '%s' namespace", pkgName, pkgVersion, packageAvailableOp.Namespace)
```

Comment:
This should not be an error. Please change it to a warning.

Reply:
changed to warning